test/e2e/framework/framework.go

```diff
@@ -69,6 +70,10 @@ func GetResource(client *ClientSet, resource string) (runtime.Object, error) {
 		res, err = GetConsoleService(client)
 	case "Route":
 		res, err = GetConsoleRoute(client)
+	case "OCCLIDownloads":
```

Comment:
This is where I wish we had taken `name` as a param as well, instead of creating `OCCLIDownloads` and `ODOCLIDownloads` as new magic strings to imply 2 different resources of the same kind.